ipaclient/install/client.py

```diff
@@ -2856,6 +2849,19 @@ def _install(options):
     if not ca_enabled:
         disable_ra()
 
+    try:
+        result = api.Backend.rpcclient.forward(
+            'config_show',
+            raw=True,  # so that servroles are not queried
+            version=u'2.0'
+        )
+    except Exception as e:
+        raise ScriptError(
```

Comment:
Could you please add a ``log.debug('msg', exc_info=True)`` line, so the exception traceback gets logged in the debug log?

Reply:
Done